vimiv/gui/metadatawidget.py

```diff
@@ -54,20 +55,42 @@ def __init__(self, parent):
             self._mainwindow_bottom = 0
             self._mainwindow_width = 0
             self._path = ""
-            self._loaded = False
+            self._current_set = ""
```

Comment:
I like the logic here a lot! Could you please also update the class docstring `Attributes` section to reflect the change?